config/distribusjon/dev-gcp.yml

```diff
@@ -1,4 +1 @@
 kafkaPool: nav-dev
-azure:
```

Comment:
Det er sikkert riktig, men hvordan vet du / fant ut at azure ikke er nødvendig for disse?

Reply:
De har kun verdiene i dev.